docs/leaderboard/develop/api-reference/index.md

```diff
@@ -0,0 +1,193 @@
+
+</details>
+
+<SdkExampleTabs snippetId={'API_LeaderboardUpsert'} />
+
+For large leaderboards, you will need to upsert in batches of 8192 elements.
+
+<SdkExampleTabs snippetId={'API_LeaderboardUpsertPagination'} />
+
+### Fetch elements by score
+
+Fetches elements that fall within the specified min and max scores.
+
+
+| Name         | Type                | Description                                                       |
+|--------------|---------------------|-------------------------------------------------------------------|
+| minScore     | Optional[number]    | Inclusive lower bound for the score range. Defaults to `-inf`.    |
+| maxScore     | Optional[number]    | Exclusive upper bound for the score range. Defaults to `+inf`.    |
+| order        | Optional[Ascending / Descending]    | The order to fetch the elements in. Defaults to ascending, meaning 0 is the lowest-scoring rank.   |
+| offset       | Optional[number]    | The number of elements to skip before returning the first element. Defaults to 0. Note: this is not the score of the first element to return, but the number of elements of the result set to skip before returning the first element.    |
+| count        | Optional[number]    | The maximum number of elements to return. Defaults to 8192, which is the maximum that can be fetched per request.    |
+
+<details>
+  <summary>Method response object</summary>
+
+* Success
+    * values(): {`id`: number, `score`: number, `rank`: number}[]
+* Error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+<SdkExampleTabs snippetId={'API_LeaderboardFetchByScore'} />
+
+For large leaderboards, you will need to fetch in batches of 8192 elements.
+
+<SdkExampleTabs snippetId={'API_LeaderboardFetchByScorePagination'} />
+
+### Fetch elements by rank
+
+Fetches elements that fall within the specified min and max ranks.
+
+
+| Name         | Type                | Description                                                                  |
+|--------------|---------------------|------------------------------------------------------------------------------|
+| startRank    | number    | Inclusive lower bound for the rank range. Defaults to 0.                     |
+| endRank      | number    | Exclusive upper bound for the rank range. Defaults to `startRank` + 8192.    |
+| order        | Optional[Ascending / Descending]    | The order to fetch the elements in. Defaults to ascending, meaning 0 is the lowest-scoring rank.   |
+
+<details>
+  <summary>Method response object</summary>
+
+* Success
+    * values(): {`id`: number, `score`: number, `rank`: number}[]
+* Error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+<SdkExampleTabs snippetId={'API_LeaderboardFetchByRank'} />
+
+For large leaderboards, you will need to fetch in batches of 8192 elements.
+
+<SdkExampleTabs snippetId={'API_LeaderboardFetchByRankPagination'} />
+
+### Fetch elements' rank
```

Comment:
This does not look like the correct name for this operation. Maybe `Fetch elements by id`?